sklearn/metrics/_regression.py

```diff
@@ -467,6 +468,10 @@ def mean_squared_log_error(
         'uniform_average' :
             Errors of all outputs are averaged with uniform weight.
 
+        'squared' : bool, default=True
+        If True returns MSLE(mean squared log error) value. 
+        If False returns RMSLE(root mean squared log error) value.
```

Comment:
Indentation is off here. Have a look at the previous parameters to see what needs to be done.

Reply:
Yes, I figured out this indentation problem and now it's fixed.